src/components/timeRangePicker/TimeRangePicker.tsx

```diff
@@ -130,6 +190,61 @@ export function TimeRangePicker({
     onChange({ start: timeStart, end: timeEnd });
   };
 
+  if (type === "Days") {
```

Comment:
```suggestion
  if (type === RANGE_TYPE.DAY) {
```

Reply:
RANGE_TYP is a different enum, but I've created another one.